### PR TITLE
Update r_basics_intro

### PR DESCRIPTION
**Pull request by pm0kjp**
I updated the R Basics: Introduction module.   This PR should be checked and then merged AFTER pull #412 .

To see if I've broken anything, look at : https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/joy-update-r-intro/r_basics_introduction/r_basics_introduction.md#1 

Basically this is what I did: 

* load bearing whitespace (still can't get bullets to work inside a div, ugh)
* use macros (overview, feedback, setup)
* add metadata

### CODE REVIEW

File: r_basics_introduction/r_basics_introduction.md [hunk 1]
```diff
@@ -19,122 +22,63 @@ After completion of this module, learners will be able to:
...
-				$img    = $this.children('img'),
-				$imgSrc = $img.attr('src'),
-				$imgAlt = $img.attr('data-alt'),
-				$imgExt = $imgAlt.split('.');
+good_first_module: true
+data_task: data_analysis
+coding_required: true
+coding_level: basic
+coding_language: r
+sequence_name: r_basics
```
Comment (rosemm) [inline r_basics_introduction/r_basics_introduction.md:40]:
```suggestion
sequence_name: r_basics
@sets_you_up_for

- r_basics_transform_data
- r_basics_visualize_data
- r_missing_values
- r_practice
- r_reshape_lonog_wide
- r_summary_stats
- data_visualization_in_ggplot2

@end
@depends_on_knowledge_available_in

@end
```

File: r_basics_introduction/r_basics_introduction.md [hunk 2]
```diff
@@ -19,122 +22,63 @@ After completion of this module, learners will be able to:
...
-This course is designed for brand new beginners with zero or minimal experience working with R.
-
-**Learning Objectives**
-
-@learning_objectives
-
-</div>
+@overview
 
 Material for this module was adapted, with permission, from [Stephan Kadauke's R for Clinical Data workshop materials](https://skadauke.github.io/intro-to-r-for-clinicians-chop/).  We owe special thanks to Dr. Kadauke as well as the R User Group at Children's Hospital of Philadelphia for their generosity in sharing these materials.
```
Comment (rosemm) [inline r_basics_introduction/r_basics_introduction.md:59]:
```suggestion
<div class = "gratitude">
<b style="color: rgb(var(--color-highlight));">Thank you!</b><br>

Material for this module was adapted, with permission, from [Stephan Kadauke's R for Clinical Data workshop materials](https://skadauke.github.io/intro-to-r-for-clinicians-chop/).  We owe special thanks to Dr. Kadauke as well as the R User Group at Children's Hospital of Philadelphia for their generosity in sharing these materials.

</div>
```

We have a highlight box for this :)

**Review by rosemm — CHANGES_REQUESTED** [review]
Missing two metadata fields, and there's highlight box formatting to add for the thank-you to Stephan/CHOPR, but other than that looks good!

**Comment by pm0kjp** [comment]
@rosemm tysm for the eagle eyes!  I committed your changes and will keep my eyes peeled on the other modules for this stuff too!

**Review by rosemm — APPROVED** [review]
lgtm!

**Pull request merged**